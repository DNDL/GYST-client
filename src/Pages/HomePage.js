import React from 'react';
import PropTypes from 'prop-types';
import UserDisplay from '../components/User/UserDisplay';
import AllHabits from '../components/Containers/AllHabits';
import Nav from '../components/Nav/Nav';
import Logo from '../components/Logo/Logo';

function HomePage() {
  return (
    <>
      <Logo />
      <UserDisplay />
      {/* {props.match.params.id ? <FriendDisplay friend={props.match.params.id} /> : <UserDisplay/> } */}
      <AllHabits/>
      <Nav />
    </>
  );
}

HomePage.propTypes = {
  props: PropTypes.shape({
    match: PropTypes.shape({
      params: PropTypes.shape({
        id: PropTypes.string.isRequired
      }).isRequired
    }).isRequired
  })
};

export default HomePage;

