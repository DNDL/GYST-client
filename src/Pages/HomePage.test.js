import React from 'react';
import { shallow } from 'enzyme';
import HomePage from './HomePage';

describe('HomePage component', () => {
  it('renders HomePage', () => {
    const wrapper = shallow(<HomePage />);
    expect(wrapper).toMatchSnapshot();
  });
});
