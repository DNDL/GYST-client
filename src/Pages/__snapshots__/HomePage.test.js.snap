// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage component renders HomePage 1`] = `
<Fragment>
  <Logo />
  <UserDisplay />
  <Connect(AllHabits) />
  <Nav />
</Fragment>
`;
